python/bids_import.py

```diff
@@ -216,6 +216,20 @@ def read_and_insert_bids(bids_dir, config_file, verbose, createcand, createvisit
                     loris_bids_root_dir    = loris_bids_root_dir
                 )
 
+            elif modality == 'ieeg':
+                Ieeg(
```

Comment:
any chance this could be all lowercase instead of `Ieeg` ?

Reply:
It could, hwever since it's a class, the convention is to have it initialized in upper case, see [this line](https://github.com/aces/Loris-MRI/blob/main/python/bids_import.py#L220) for example. Let me know if you would still prefer to make it all lowercase.